test/test.rs

```diff
@@ -13,12 +13,7 @@ mod test_errno;
 mod test_fcntl;
 #[cfg(linux_android)]
 mod test_kmod;
-#[cfg(any(
-    freebsdlike,
-    target_os = "fushsia",
```

Comment:
Before this PR, this test module was enabled for target `fushsia`(note it is a typo), it does not exist, and since module `mqueue` does not exist on Fuchsia, simply remove this line